diff --git a/export/rspec-rails-RSpec-Rails.current b/export/rspec-rails-RSpec-Rails.new
index 56ca611..dd2abcc 100644
--- a/export/rspec-rails-RSpec-Rails.current
+++ b/export/rspec-rails-RSpec-Rails.new
@@ -22,6 +25,15 @@ Module: RSpec::Rails
 lib/rspec-rails.rb
 
   
+    Defined in:
+    lib/rspec-rails.rb
+
+  
+  
+
+  
+
+  
   
   
   
@@ -84,7 +96,17 @@ Defined Under NamespaceDefined Under Namespace
   
 
 
-  Constant SummaryConstant Summary
+  
+    
+      Constant Summary
+      
+    
+      Constant Summary
+      
+    
+
+    
+      
         
         
       
